license_check.py

```diff
@@ -35,15 +36,15 @@ def check_copyright(filename, copyright_notice):
             if copyright_notice in line:
                 return True
 
-        eprint(f'No copyright notice found in {filename}')
+        eprint(f'Copyright not found in ...\t\t{filename}')
```

Comment:
I don't get the need to change the phrasing and the need for  `...\t\t`

Reply:
When I perform the checker on the partitioner I got several errors. The format has very messy to understand, that 's why I  tried to reformat the text to be more identical to license output in number of characters. And the ´\t\t´ is to line up the file paths with the license print.

Reply:
I tried it myself and do agree. I'd suggest even simpler:

```suggestion
        eprint(f'Copyright not found: \t\t{filename}')
```

Reply:
Changed.

Reply:
@josecm Kindly resolve this conversation if the changes attend to your comments. Thank you.